src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAccountTransferAllowance.sql

```diff
@@ -0,0 +1,30 @@
+﻿CREATE PROCEDURE [employer_financial].[GetAccountTransferAllowance]
```

Comment:
The transfer allowance needs to be calculated once a year and stored as late declarations and adjustments shouldn't be taken into consideration.

Reply:
But payments will affect this values so we are going to need to on demand unless we change it every time a payment transaction is added to the account?

Reply:
Yes, following our discussion yesterday my comment was rubbish!

Reply:
i'm going to leave this issue as it is as we will cover this when we do the payments work. This code will not be released until then as its part of the feature. How we manage payments will be the main affect of how we cache/store pre calculated values.